apps/condo/domains/marketplace/components/MarketplacePaymentsContent.tsx

```diff
@@ -141,15 +153,21 @@ const MarketplacePaymentsTableContent = () => {
         fetchPolicy: 'network-only',
     })
 
-    const { data: allPaymentsSum, loading: allPaymentsSumLoading } = usePaymentsSum({ ...searchPaymentsQuery })
-    const { data: donePaymentsSum, loading: donePaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status: PAYMENT_DONE_STATUS })
-    const { data: inProcessPaymentsSum, loading: inProcessPaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_PROCESSING_STATUS, PAYMENT_WITHDRAWN_STATUS, PAYMENT_ERROR_STATUS] })
+    const { data: allPaymentsSum, loading: allPaymentsSumLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [ PAYMENT_WITHDRAWN_STATUS, PAYMENT_DONE_STATUS] })
+    const { data: donePaymentsSum, loading: donePaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_DONE_STATUS] })
+    const { data: inProcessPaymentsSum, loading: inProcessPaymentsLoading } = usePaymentsSum({ ...searchPaymentsQuery, status_in: [PAYMENT_WITHDRAWN_STATUS] })
 
     const [selectedRows, setSelectedRows] = useState([])
 
     const [search, handleSearchChange] = useSearch()
     const handleSearch = useCallback((e) => {handleSearchChange(e.target.value)}, [handleSearchChange])
 
+    const [dateRange, setDateRange] = useDateRangeSearch('createdAt')
+
+    const disabledDate = useCallback((currentDate) => {
+        return currentDate && currentDate < dayjs().startOf('year')
+    }, [])
```

Comment:
This way you will not be able to look at payments for the previous year. 
For example, today is January. And you won't be able to select December of the previous year or any other month - this could be critical

A few days ago we fixed this problem on the analytics page. You can use something like this:
```
    const disabledDate = useCallback((currentDate) => {
        const minDate = dayjs().startOf('year').subtract(1, 'year')
        const maxDate = dayjs().endOf('year')
        return currentDate && (currentDate < minDate || currentDate > maxDate)
    }, [])
```